### PR TITLE
*: update error code.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
@shenli  @coocood  PTAL

File: executor/executor.go [hunk 1]
```diff
@@ -60,11 +60,11 @@ var (
 
 // Error codes.
 const (
-	CodeUnknownPlan terror.ErrCode = iota
-	CodePrepareMulti
```
Comment (shenli) [inline executor/executor.go]: We need to specify type for each var?

Comment (coocood) [inline executor/executor.go]: Otherwise the type is going to be `int`.

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Pull request merged**